db/database.go

```diff
@@ -50,9 +50,13 @@ const (
 )
 
 const (
-	DefaultRevsLimitNoConflicts             = 50
-	DefaultRevsLimitConflicts               = 100
-	DefaultPurgeInterval                    = 30 // Default metadata purge interval, in days.  Used if server's purge interval is unavailable
+	DefaultRevsLimitNoConflicts = 50
+	DefaultRevsLimitConflicts   = 100
+
+	// DefaultPurgeInterval represents a time duration of 30 days (in hours)
```

Comment:
`time.Duration` is measured in nanoseconds, but it's not relevant enough to be included in the comment.

```suggestion
	// DefaultPurgeInterval represents a time duration of 30 days
```

Reply:
This comment has been revised in latest commit.